pkg/csi/service/common/vsphereutil.go

```diff
@@ -645,3 +646,40 @@ func isExpansionRequired(ctx context.Context, volumeID string, requestedSize int
 
 	return currentSize < requestedSize, nil
 }
+
+// CreateSnapshotUtil is the helper function to create CNS snapshot for given volumeId
+// the desc parameter denotes the snapshot description required by CNS CreateSnapshot API.
+// 	  The CreateSnapshotRequest Name is expected for this parameter. The default format is, "snapshot-<UUID>".
+// the returned string is the unique snapshotID generated by the storage system, i.e., CNS
```

Comment:
This is confusing.  The returned string parameter is volume id + snapshot id, not just the snapshot id?